google/auth/identity_pool.py

```diff
@@ -117,28 +120,65 @@ def __init__(
                     )
             else:
                 self._credential_source_field_name = None
-        else:
-            self._credential_source_file = None
-        if not self._credential_source_file:
-            raise ValueError("Missing credential_source file")
+
+        if self._credential_source_file and self._credential_source_url:
+            raise ValueError("Ambiguous credential_source")
+        if not self._credential_source_file and not self._credential_source_url:
+            raise ValueError("Missing credential_source")
```

Comment:
Please make this more explicit about what field(s) are causing the error

Examples:

- `Ambiguous credentials_source. 'file' is mutually exclusive with 'url'.`

- `Missing credentials_source. A 'file' or 'url' must be provided.`

Reply:
Done